jetson/nav/search/searchStateMachine.cpp

```diff
@@ -284,28 +285,86 @@ NavState SearchStateMachine::executeSearchDrive( Rover* phoebe, const rapidjson:
 } // executeSearchDrive()
 
 // Executes the logic for turning to the target.
+// Starts by turnign the gimbal towards the target. Then holds for 200 cycles to confirm aquisition
```

Comment:
Instead of 200 cycles, this could be done using the perception wait time defined in the config, it would help with standardization and using meaningful units